unime/src/i18n/en/index.ts

```diff
@@ -189,26 +194,64 @@ const en = {
       DATA: 'Data',
       BADGES: 'Badges',
     },
+    CREDENTIAL: {
+      NAVBAR_TITLE: 'Credential info',
+      NAME: '',
+    },
+    BADGE: {
+      NAVBAR_TITLE: 'Badge info',
+      DESCRIPTION: 'Description',
+      METADATA: 'Metadata',
+    },
     EMPTY_CREDENTIALS: {
       TITLE: "It's a bit quiet in here",
       SUBTITLE: 'Why not add some credentials to start your new digital me?',
     },
     FAVORITES: 'My favorites',
   },
   ACTIVITY: {
+    BOTTOM_NAVIGATION_TITLE: 'Activity',
     NAVBAR_TITLE: 'Connected',
     TABS: {
       CONNECTIONS: 'Connections',
       TIMELINE: 'Timeline',
     },
   },
+  SCAN: {
+    BOTTOM_NAVIGATION_TITLE: 'Scan',
+    TITLE: 'Scan a',
+    TITLE1: 'QR Code',
+    SUBTITLE: 'Bring a QR Code into view of this screen to start an interaction.',
+    PERMISSION: 'No permissions to',
+    PERMISSION1: 'access the camera',
+    LABELS: {
```

Comment:
These `LABELS` are purely for us developers and thus do not need to be translated as they will never appear to the user. Sorry, I should've been more clear about this.